Tests/Marketplace/approved_usecases.json

```diff
@@ -0,0 +1,27 @@
+[
+	"Access",
+	"Breach Notification",
+	"Breach and Attack Simulation",
+	"Brute Force",
+	"Case Management",
+	"Compliance",
+	"Crisis Management",
+	"Default",
+	"Email Communication",
+	"Employee Offboarding",
+	"GDPR Breach Notification",
+	"HIPAA Breach Notification",
+	"Health Check",
+	"Hunting",
+	"Identity And Access Management",
```

Comment:
"Identity And Access Management" or "Identity Lifecycle Management"?
@glicht @michalgold

Reply:
The use case is "Identity And Access Management"